### PR TITLE
Add content about the Alibaba cloud provider

### PR DESCRIPTION
**Pull request by gujingit**
Add content about the Alibaba cloud provider

### CODE REVIEW

File: content/zh/docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -479,7 +479,7 @@ The following cloud providers have implemented CCMs:
 * [OpenStack](https://github.com/kubernetes/cloud-provider-openstack)
 * [Oracle](https://github.com/oracle/oci-cloud-controller-manager)
 -->
-
+* [Alibaba Cloud](https://github.com/kubernetes/cloud-provider-alibaba-cloud)
```
Comment (tanjunchen) [inline content/zh/docs/concepts/architecture/cloud-controller.md:482]: /LGTM
for zh

**Comment by jimangel** [comment]
/lgtm

Could I get an `/approve` from one of the @kubernetes/sig-docs-zh-owners?

**Pull request merged**